v1/api/swagger/paths/services.yaml

```
@@ -5,11 +5,15 @@ get:
   description: >-
     Returns an array of all OS service images objects that have been created.
     If GreenLake IAM issued token is used for authentication, 
```

Maybe do a global grep for "GreenLake IAM" string & make sure they are all replaced with "GreenLake Cloud Services IAM".

```suggestion
    If GreenLake Cloud Services IAM issued token is used for authentication, 
```